.github/workflows/ci.yaml

```
@@ -74,4 +72,4 @@ jobs:
     - name: Build Nix dependencies
       run: nix-shell --pure --run "echo '=== Nix dependencies installed ==='"
     - name: Build
-      run: nix-shell --pure --run "stack build --pedantic --test --bench --no-run-tests --no-run-benchmarks"
+      run: nix-shell --pure --run "stack build --pedantic --test --bench"
```

We should probably not run benchmarks here. I'm not sure what purpose they would serve.
```suggestion
      run: nix-shell --pure --run "stack build --pedantic --test --bench --no-run-benchmarks"
```